lib/CL/pocl_builtin_kernels.c

```diff
@@ -832,10 +840,31 @@ pocl_validate_dbk_attributes (BuiltinKernelId kernel_id,
     case POCL_CDBI_DBK_EXP_JPEG_DECODE:
     case POCL_CDBI_DBK_EXP_JPEG_ENCODE:
       return pocl_validate_khr_jpeg (kernel_id, kernel_attributes);
-    default:
+#ifdef HAVE_ONNXRT
+    case POCL_CDBI_DBK_EXP_ONNX_INFERENCE:
+      {
```

Comment:
Can we move the arg validation also to this file?  Also please don't use C++ comments in C.

Reply:
should be possible, there is already a validate attributes function